spec/search-api.yaml

```diff
@@ -88,6 +96,24 @@ paths:
           description: This server does not implement the search operation
         '500':
           description: An unexpected error occurred
+  /service-info:
+    get:
+      summary: 'Show information about the Search service'
+      operationId: getServiceInfo
+      tags:
+        - Service Info API
```

Comment:
As noted above the service info API appears to use the tag 'service-info' ... I would think we should standardize these in some way but failing that we should probably at least be consistent.